### PR TITLE
Default parameter assignment is not supported in IE

### PR DESCRIPTION
**Pull request by DBezemer**
Default assignment will be supported from ECMA2015 and onwards, which IE does not (yet) use
http://www.ecma-international.org/ecma-262/6.0/#sec-functiondeclarationinstantiation

This currently breaks the Admin Console functions in IE

### CODE REVIEW

File: admin_console/views/scripts/partner/list.phtml [hunk 1]
```diff
@@ -462,10 +462,10 @@ function assignLiveDeliveryProfile(format, currentDps)
 	);
 }
 
-function assignDeliveryProfile(format, currentDps, type = null)
+function assignDeliveryProfile(format, currentDps, type)
 {
+	var name = name === undefined? null: name;
```
Comment (yossipapi) [inline admin_console/views/scripts/partner/list.phtml]: This var is not used so why was it added?
Does name = type?

**Comment by DBezemer** [comment]
@yossipapi brain fart, obviously name has to be type, changed it

File: admin_console/views/scripts/partner/list.phtml (hunk 1 above)
Comment (DBezemer) [inline admin_console/views/scripts/partner/list.phtml]: Yes, changed the commit

**Comment by DBezemer** [comment]
@yossipapi can you please confirm / merge? Without this Admin Console does not work in IE. As it's now branch 12.18 let me know if you want me to reopen a pull against that branch

**Pull request merged**